src/FixedDataTableRow.js

```diff
@@ -190,6 +202,21 @@ class FixedDataTableRowImpl extends React.Component {
     return width;
   };
 
+  _renderRowExpanded = (/*number*/ cellHeight) => /*?object*/ {
```

Comment:
I wonder if we should include this styling in the repo at all.  It may be more flexible if we just render the rowExpanded prop directly.  That probably means passing the rowExpandedGetter & index into this row though and passing the width & height into rowExpandedGetter.  Thoughts?